content/guide/dsl-spec-plugins.md

```diff
@@ -31,34 +31,59 @@ plugins:
 
 ## Plugin Definition
 
-Keyname           | Required    | Type        | Description
------------       | --------    | ----        | -----------
-executor          | yes         | string      | Where to execute the plugin's operations. Valid Values: `central_deployment_agent`, `host_agent`.
-source            | conditional | string      | Where to retrieve the plugin from. Could be either a path relative to the `plugins` dir inside the blueprint's root dir or a url. If `install` is `false`, `source` is redundant. If `install` is true, `source` is mandatory.
-install_arguments | no          | string      | Optional arguments passed to the 'pip install' command created for the plugin installation
-install           | no          | boolean     | Whether to install the plugin or not as it might already be installed as part of the agent. Defaults to `true`.
+Keyname              |   Required  | Type        | Description
+-----------          | --------    | ----        | -----------
+executor             | yes         | string      | Where to execute the plugin's operations. Valid Values: `central_deployment_agent`, `host_agent`. See [Plugin Executor](#executor)
+source               | conditional | string      | Where to retrieve the plugin from. Could be either a path relative to the `plugins` dir inside the blueprint's root dir or a url. If `install` is `false`, `source` is redundant. If `install` is true, `source` (or `pacakge_name`) is mandatory. See [Source Plugins](#source-plugins)
+install_arguments    | no          | string      | Optional arguments passed to the 'pip install' command created for the plugin installation
+install              | no          | boolean     | Whether to install the plugin or not as it might already be installed as part of the agent. Defaults to `true`. (Supported since: [cloudify_dsl_1_1](dsl-spec-versioning.html))
+package_name         | no          | string      | Managed plugin package name. See [Managed Plugins](#managed-plugins). (Supported since: [cloudify_dsl_1_2](dsl-spec-versioning.html)) If `install` is `false`, `pacakge_name` is redundant. If `install` is true, `package_name` (or `source`) is mandatory.
+package_version      | no          | string      | Managed plugin pacakge version. See [Managed Plugins](#managed-plugins). (Supported since: [cloudify_dsl_1_2](dsl-spec-versioning.html))
+supported_platform   | no          | string      | Managed plugin supported platform. See [Managed Plugins](#managed-plugins). (Supported since: [cloudify_dsl_1_2](dsl-spec-versioning.html))
```

Comment:
after discussing with near, it turns out the output is different.
it will be something along the lines of `linux_x86_64`, i'm currently using the same function used by wagon to implicitly decide the current platform `wagon.utils.get_platform()`